.github/maintainers_guide.md

```diff
@@ -0,0 +1,93 @@
+# Maintainers Guide
+
+This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain
+this project. If you use this package within your own software as is but don't plan on modifying it, this guide is
+**not** for you.
+
+## Tools
+
+All you need to work with this project is a supported version of [Node.js](https://nodejs.org/en/)
+(see `package.json` field "engines") and npm (which is distributed with Node.js).
+
+## Tasks
+
+### Testing
+
+This package has unit tests for most files in the same directory the code is in with the suffix `.spec` (i.e. `exampleFile.spec.ts`). You can run the entire test suite using the npm script `npm test`. This command is also executed by Travis, the continuous integration service, for every Pull Request and branch. The coverage is computed with the `codecode` package. The tests themselves are run using the `mocha` test runner.
```

Comment:
Considering the following statement, 
>Test code should be written in syntax that runs on the oldest supported Node.js version, without transpiling.

the example file name `exampleFile.spec.ts` needs to be `exampleFile.spec.js`.

Reply:
Ah, no. All the tests in this project are written in TS. Therefore, just removing the part `, without transpiling` would be fine. 👍